py3status/modules/wifi.py

```diff
@@ -63,16 +64,44 @@ class Py3status:
     cache_timeout = 10
     device = 'wlan0'
     down_color = 'bad'
-    format_down = 'W: down'
-    format_up = 'W: {bitrate} {signal_percent} {ssid}'
+    format = 'W: [{bitrate} {signal_percent} {ssid}|\?show down]'
```

Comment:
we can do `W: [{bitrate} {signal_percent} {ssid}|\?show down]` as the format if we do a trick like in getjson, it's that little bit cleaner.

`if ssid: data = {all the data} else data = None` then `safe_format(format, data)` type idea but done nicely